README.md

```diff
@@ -1,15 +1,147 @@
 # QX Simulator
 
-A universal quantum computer simulator. Allows simulation of a quantum circuit written in [cQasm](https://github.com/QuTech-Delft/libqasm).
+[![CI](https://github.com/QuTech-Delft/qx-simulator/workflows/Test/badge.svg)](https://github.com/qutech-delft/qx-simulator/actions)
+[![PyPI](https://badgen.net/pypi/v/qxelarator)](https://pypi.org/project/qxelarator/)
 
-Runs as a [Python module](https://pypi.org/project/qxelarator/) or an executable binary.
+A universal quantum computer simulator. Allows simulation of a quantum circuit written in [cQASM](https://qutech-delft.github.io/cQASM-spec/) v3.0.
 
-For detailed documentation, please visit [the ReadTheDocs page](https://qx-simulator.readthedocs.io/en/latest/), to find e.g.:
-- [installation instructions](https://qx-simulator.readthedocs.io/en/latest/manual/installation.html)
-- [usage](https://qx-simulator.readthedocs.io/en/latest/manual/usage.html).
+Can be used as a [Python module](https://pypi.org/project/qxelarator/) or as an executable.
+
+## User manual
+
+[Coming soon] QX Simulator documentation is hosted through GitHub Pages [here](https://QuTech-Delft.github.io/qx-simulator/).
+
+## File organization
+
+For development, see:
+
+- `include`: public headers.
+- `src`: source files.
+- `test`: test files.
+- `python`: SWIG interface.
+
+For build process, and continuous integration:
+
+- `.github`: GitHub Actions configuration files.
+- `conan`: Conan profiles.
+
+Build outputs may go into:
+
+- `build/<build type>`: the C++ library output files generated by Conan.
+- `pybuild`: the Python library output files generated by `setup.py`.
+
+## Dependencies
+
+* C++ compiler with C++23 support (gcc 12, clang 15, msvc 19)
+* `CMake` >= 3.12
+* `git`
+* `Python` 3.x plus `pip`, with the following package:
+  * `conan` >= 2.0.6
+* `SWIG`
+
+## Build
+
+This version of QX Simulator can only be compiled via the Conan package manager.
+You'll need to create a default profile before using it for the first time.
+
+The installation of dependencies, as well as the compilation, can be done in one go.
```

Comment:
The installation and compilation of the dependencies can be done in one go

Reply:
@juanboschero for these type of small changes, you can suggest yourself a code change by using the
<img width="13" alt="image" src="https://github.com/user-attachments/assets/9d566fe2-b5e2-46b8-bb05-cf4a50c8871d"> buttton.